m3-sys/cm3ide/src/nodes/Source.m3

```diff
@@ -23,11 +23,10 @@ VAR
   clientID := ID.Add ("clients");
 
 CONST
-  Kind = { I3, M3, IG, MG, C, CPP, H, Quake, Other };
   IsM3 = ARRAY Kind OF BOOLEAN { TRUE,  (* i3 *)
                                  TRUE,  (* m3 *)
                                  TRUE,  (* ig *)
-                                 TRUE,  (* mg )
+                                 TRUE,  (* mg *)
```

Comment:
Oops, thank you. I am not sure I needed to change cm3ide.
Maybe revert, just around here?

Reply:
} Oops, thank you
 No problem -)
 
} I am not sure I needed to change cm3ide.
} Maybe revert, just around here?
 I do not shue.
 In any case "my minor fixes" are need.